x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/assets/deferred_assets_warning.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiIcon, EuiToolTip } from '@elastic/eui';
+
+export const DEFERRED_ASSETS_WARNING_LABEL = i18n.translate(
+  'xpack.fleet.packageCard.reauthorizationRequiredLabel',
+  {
+    defaultMessage: 'Reauthorization required',
+  }
+);
+
+export const DEFERRED_ASSETS_WARNING_MSG = i18n.translate(
+  'xpack.fleet.epm.packageDetails.assets.deferredInstallationsMsg',
+  {
+    defaultMessage:
+      'This package has at least one deferred installation which might require additional permissions to install and operate correctly.',
```

Comment:
Ditto re removing uncertainty:
```suggestion
      'This package has at least one deferred installation which requires additional permissions to install and operate correctly.',
```

Reply:
Updated here [`98a01d4` (#154665)](https://github.com/elastic/kibana/pull/154665/commits/98a01d409be57bc46df59fcfbe6e17c92b71c780)